src/renderer/features/governance/components/Delegations/DelegateTitle.tsx

```diff
@@ -10,5 +11,20 @@ type Props = {
 export const DelegateTitle = ({ delegate, className }: Props) => {
   const delegateTitle = delegate.name || <Truncate ellipsis="..." start={4} end={4} text={delegate.accountId} />;
 
-  return <HeadlineText className={cnTw('w-full', className)}>{delegateTitle}</HeadlineText>;
+  return (
+    <HeadlineText className={cnTw('w-full', className)}>
+      <div className="inline-flex items-center">
+        {delegateTitle}{' '}
+        {delegate.address && (
+          <span>
```

Comment:
can you remove this span w/o breaking layout?

Reply:
This component is tricky. Removing the span, even with a `display: inline-block` on the `ExplorersPopover` makes such changes
Before:
![image](https://github.com/user-attachments/assets/8cc35581-9f73-4e1f-ba6a-521ecda04ed5)

After:
![image](https://github.com/user-attachments/assets/2878b292-c0bb-474f-986a-b611493908cb)